patroni/postgresql/bootstrap.py

```diff
@@ -127,7 +127,7 @@ def error_handler(e: str) -> None:
         if self._postgresql.config.superuser:
             if 'username' in self._postgresql.config.superuser:
                 options.append('--username={0}'.format(self._postgresql.config.superuser['username']))
-            if 'password' in self._postgresql.config.superuser:
+            if 'password' in self._postgresql.config.superuser and self._postgresql.config.superuser['password']:
```

Comment:
I wonder if we should support the case when password is an empty string.
```suggestion
            if isinstance(self._postgresql.config.superuser.get('password') , str):
```

Actually, --validate-config successfully catching missing password value:
```
postgresql.authentication.superuser.password None is not a string
```

Reply:
Hrm, I kept it that way since https://github.com/patroni/patroni/issues/928 cause not having `password:` there back then did not work, but I guess I should have changed it since #930 got merged. I will at least change it in the Debian package now.

I should really use `patroni --validate-config` more often...